cinema/urls.py

```diff
@@ -0,0 +1,25 @@
+"""cinema URL Configuration
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/4.0/topics/http/urls/
+Examples:
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  path('', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+from django.contrib import admin
+from django.urls import path, include
+
+urlpatterns = [
+    path('admin/', admin.site.urls),
+    path('', include('basepage.urls')),
+    path('users/', include('django.contrib.auth.urls')),
+    path('users/',include('app_users.urls')),
```

Comment:
You might get some unwanted behavior if you are going to use the same path for multiple url files.
Similar to what might happen by adding here all the paths without a prefix.